pages/1.13/installing/production/advanced-configuration/configuration-reference/index.md

```diff
@@ -587,6 +589,13 @@ The SSH username, for example `centos`.
 
 **Note:** The ssh_user is not required for the production installer.
 
+### superuser_service_account_public_key
+The superuser service account public key. The operator is advised to create a RSA key-pair and supply the generated public key in X.509 SubjectPublicKeyInfo/OpenSSL PEM public key format.
+This additional superuser account is used to provide a method of automating tasks against a DC/OS cluster with superuser privileges.
+
+### superuser_service_account_uid
+The superuser service account user ID. This service account uses the `superuser_service_account_public_key`.
```

Comment:
1. Remove the first line `The superuser service account  user ID.`
2. Begin with:
`This service account...